server/testutils/factories/courseParticipationOutcome.ts

```diff
@@ -13,15 +13,15 @@ const outcomeTypes = {
   complete() {
     return {
       detail: faker.lorem.paragraph({ max: 5, min: 0 }),
-      status: 'complete' as CourseParticipationOutcomeStatus,
+      status: 'complete' as CourseParticipationOutcome['status'],
```

Comment:
Might be better to do to `complete(): CourseParticipationOutcome` and same for `incomplete()`.  Save having to use `as` and then the other values will be typed too.

Reply:
Good shout